fcrepo-kernel/src/main/java/org/fcrepo/kernel/exception/ManagedTypeException.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.exception;
+
+/**
+ * Certain mixin types are managed by the repository only.
+ *
+ * @author whikloj
+ * @since 2015-06-02
+ */
+public class ManagedTypeException extends ConstraintViolationException {
```

Comment:
You are correct. I misread your comment above. The two exception types seem redundant to me, but if you like them, then sure, let's keep them. At a minimum, for clarity I would suggest aligning their names to either:
* `ManagedTypeException` and `ManagedPropertyException`, or
* `ServerManagedTypeException` and `ServerManagedPropertyException`